Subject: locking/RT: Add might sleeping annotation.
From: Sebastian Andrzej Siewior <bigeasy@linutronix.de>
Date: Thu May 20 18:09:38 2021 +0200

From: Sebastian Andrzej Siewior <bigeasy@linutronix.de>

Signed-off-by: Sebastian Andrzej Siewior <bigeasy@linutronix.de>
Signed-off-by: Thomas Gleixner <tglx@linutronix.de>


---
 kernel/locking/spinlock_rt.c | 1 +
 1 file changed, 1 insertion(+)
---
diff --git a/kernel/locking/spinlock_rt.c b/kernel/locking/spinlock_rt.c
index 19a5e3baa1f0..85b19e41e26d 100644
--- a/kernel/locking/spinlock_rt.c
+++ b/kernel/locking/spinlock_rt.c
@@ -32,6 +32,7 @@ static __always_inline void rtlock_lock(struct rt_mutex *rtm)
 
 static __always_inline void __rt_spin_lock(spinlock_t *lock)
 {
+	___might_sleep(__FILE__, __LINE__, 0);
 	rtlock_lock(&lock->lock);
 	rcu_read_lock();
 	migrate_disable();
